gdx/src/com/badlogic/gdx/scenes/scene2d/utils/UIUtils.java

```diff
@@ -6,9 +6,12 @@
 import com.badlogic.gdx.Input.Keys;
 
 public class UIUtils {
-	static public boolean isMac = System.getProperty("os.name").contains("OS X");
-	static public boolean isWindows = System.getProperty("os.name").contains("Windows");
-	static public boolean isLinux = System.getProperty("os.name").contains("Linux");
+	//Adapted system checks from com.badlogic.gdx.utils.SharedLibraryLoader
+	static public boolean isAndroid = System.getProperty("java.runtime.name").contains("Android");
+	static public boolean isMac = !isAndroid && System.getProperty("os.name").contains("OS X");
+	static public boolean isWindows = !isAndroid && System.getProperty("os.name").contains("Windows");
+	static public boolean isLinux = !isAndroid && System.getProperty("os.name").contains("Linux");
+	static public boolean isIos = !isAndroid && ("iOS".equals(System.getProperty("moe.platform.name")) || !(isWindows || isLinux || isMac));
```

Comment:
If this works, which I don't have much reason to doubt, this should be a valuable addition to UIUtils.

Reply:
@tommyettinger 

https://github.com/libgdx/libgdx/blob/11a53e2d9d573e2a0a94457b309dbf7044623497/gdx/src/com/badlogic/gdx/utils/SharedLibraryLoader.java#L36-L67

Yes it works.

Reply:
Ok got something! "Mac" and  "OS X" damn.